pkgs/applications/misc/emem/default.nix

```diff
@@ -2,14 +2,14 @@
 
 stdenv.mkDerivation rec {
   pname = "emem";
-  version = "0.2.41";
+  version = "0.2.42";
   name = "${pname}-${version}";
 
   inherit jdk;
 
   src = fetchurl {
```

Comment:
ah, hmm didn't think about that :) Sorry, well if you didn't know the above method, at least you know now :)